validator/client/runner.go

```diff
@@ -172,3 +184,35 @@ func handleAssignmentError(err error, slot uint64) {
 		log.WithField("error", err).Error("Failed to update assignments")
 	}
 }
+
+func (v *validator) handleAccountsChanged(ctx context.Context, accountsChangedChan chan struct{}) {
+	validatingPubKeysChan := make(chan [][48]byte, 1)
+	var sub event.Subscription
+	importedKeymanager, ok := v.keyManager.(*imported.Keymanager)
```

Comment:
Use a switch here and throw an error or return if the keymanager is remote perhaps

Reply:
You can do a type switch
```go
switch v.keyManager.(type) {
}
```

Reply:
Thanks. totally forgot about the switch option :-)

As for throwing an error, I cannot do that. The runner will execute this function with all types of wallets, and therefore throwing an error for remote wallets would make them unusable.